lib/web_ui/lib/src/engine/skwasm/skwasm_impl/canvas.dart

```diff
@@ -18,9 +18,9 @@ class SkwasmCanvas implements SceneCanvas {
   SkwasmCanvas.fromHandle(this._handle);
   CanvasHandle _handle;
 
-  void delete() {
-    canvasDestroy(_handle);
-  }
+  // Note that we do not need to deal with the finalizer registry here, because
+  // the underlying native skia object is tied directly to the lifetime of the
+  // associated SkPictureRecorder.
```

Comment:
Good point. I wonder if in debug mode we should watch for recorders dropped on the floor and issue an error.